agent/command/s3_put.go

```diff
@@ -198,13 +201,11 @@ func (s3pc *s3put) expandParams(conf *internal.TaskConfig) error {
 		s3pc.skipMissing = false
```

Comment:
could we remove here too while we're here?

Reply:
done